crypto3/libs/blueprint/include/nil/blueprint/components/algebra/fields/plonk/addition.hpp

```diff
@@ -161,6 +161,26 @@ namespace nil {
                     component, start_row_index);
             }
 
+            template<typename BlueprintFieldType>
+            typename plonk_native_addition<BlueprintFieldType>::result_type
+                generate_assignments_(
```

Comment:
Maybe better use plonk_assignment_table  here, it's just a 'using' over plonk_table, but looks better. :)

Reply:
yes, `plonk_assignment_table` looks better)